### PR TITLE
add initial workshop time project time guidelines

### LINKED ISSUE #210: Message club leaders to ask them for feedback on pull request
Keep it brief and below 50 lines

### PR DESCRIPTION
**Pull request by jonleung**
@paked, @Bogidon, @gemmabusoni, I would love your thoughts on this!

/cc @hellyeah @zachlatta @MaxWofford @gemmabusoni @xeonjake 

fixes #210

### CODE REVIEW

File: playbook/workshop_project_model.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Club Structure
+
+People seem to be more motivated when they get to choose what they do. Ideally
+therefore, a Hack Club would be formatted like a maker space. Hack Club would be
```
Comment (MaxWofford) [inline playbook/workshop_project_model.md]: > Ideally therefore, a Hack Club would

@jonleung May want to check the order of words here.

File: playbook/workshop_project_model.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+a space for people to come in and hack on projects together. However this does
+not work for complete beginners because they don't know where to start.
+
+One solution is only to teach them the basics through lessons or workshops for
+the first several months. That's about as exciting as math teachers saying you
+have to take years of math before you get to do anything interesting with it. We
+want the beginner to be able to build their own project as soon as possible.
```
Comment (MaxWofford) [inline playbook/workshop_project_model.md]: > to teach them
> the beginner

`Beginner` starts as plural, then slips into singular

File: playbook/workshop_project_model.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Therefore we want them to learn just enough to build their own projects and then
+as soon as possible, actually build their own project. This builds confidence
```
Comment (MaxWofford) [inline playbook/workshop_project_model.md]: > Therefore, we want them to learn just enough to build their own projects and then as soon as possible, actually build their own project.

Can this be written without the repetition?

File: playbook/workshop_project_model.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+and motivation in the beginner.
+
+At [Hack Camp](https://camp.hackedu.us/), we repeated a 2-day cycle where:
+
+1. learners spend the first 2 hours of their first day following a workshop
+2. for the remainder of the first day and all of the second day, they would
```
Comment (MaxWofford) [inline playbook/workshop_project_model.md]: This ordered list reads like a sentence. What if the numbering were taken out entirely?

File: playbook/workshop_project_model.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+   come up with and build their own projects
+[the details are documented here](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/SCHEDULE.md).
```
Comment (MaxWofford) [inline playbook/workshop_project_model.md]: Should this  line be part of the sentence above this line?

File: playbook/workshop_project_model.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+We think a similar division can work for clubs:
+
+**Workshop Time**
+
+Have one club meeting where a learner learns to ship a project by following
```
Comment (MaxWofford) [inline playbook/workshop_project_model.md]: > Have one club meeting where a beginning hacker learns to

File: playbook/workshop_project_model.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+a tutorial
+
+**Project Time**
+Let them work on their own project the 1-3 club meetings before starting
```
Comment (MaxWofford) [inline playbook/workshop_project_model.md]: > et them work on their own project at the 1-3 club meetings before starting

**Comment by jonleung** [comment]
fixes #210

File: playbook/workshop_project_model.md (hunk 1 above)
Comment (jonleung) [inline playbook/workshop_project_model.md]: :+1:

File: playbook/workshop_project_model.md (hunk 2 above)
Comment (jonleung) [inline playbook/workshop_project_model.md]: :+1:

File: playbook/workshop_project_model.md (hunk 3 above)
Comment (jonleung) [inline playbook/workshop_project_model.md]: :+1:

File: playbook/workshop_project_model.md (hunk 4 above)
Comment (jonleung) [inline playbook/workshop_project_model.md]: I think it's clearer as a list

File: playbook/workshop_project_model.md (hunk 5 above)
Comment (jonleung) [inline playbook/workshop_project_model.md]: :+1:

File: playbook/workshop_project_model.md (hunk 6 above)
Comment (jonleung) [inline playbook/workshop_project_model.md]: :+1:

File: playbook/README.md [hunk 8]
```diff
@@ -7,3 +7,8 @@ The playbook will eventually also cover general facilitation guidelines. For
 now, many clubs are creating [case studies](../case_studies) of what they're
 doing in each meeting. Once patterns become clear for what's working across
 clubs, those patterns will be centralized in the playbook.
+
+## A preliminary clubs model
```
Comment (jonleung) [inline playbook/README.md]: This is a sample comment. Please ignore.

**Comment by jonleung** [comment]
@MaxWofford I fixed the changes you suggested but before we merge, I'd love to get the feedback of the Lab Directors!


@AbigailMed @amysorto @Bogidon @gemmabusoni @JevinSidhu @paked @sneharamapo @ssun @tejasmanohar @temucher @venethefifth and @xeonjake I'd love your feedback on breaking club meetings into workshop time and project time. I've talked about this to @Bogidon and he feels more comfortable doing a higher ratio of workshop times to project times.

To leave comments that reference a specific line:

Click "Files changed" at the top:
> image

And then click on the line number to comment:
> 

You can also respond to each other's comments! It would be cool to see if a discussion can happen here on GitHub!

/cc @zachlatta @hellyeah

File: playbook/workshop_project_model.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
+# Club Structure
+
+People seem to be more motivated when they get to choose what they do.
+Therefore, a Hack Club would ideally be formatted like a maker space. Hack Club
```
Comment (paked) [inline playbook/workshop_project_model.md]: *playing devils advocate*

What is a maker space?

File: playbook/workshop_project_model.md (hunk 4 above)
Comment (paked) [inline playbook/workshop_project_model.md]: List needs capitalisation

File: playbook/workshop_project_model.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+
+## Workshop Time F.A.Q
+
+### So should my workshops be lecture based?
+
+Most likely not. People learn differently and at different rates. If you just
+talk at them, that might work for one person in the room but everyone else will
+need you to go faster and slower.
+
+There should be no point ever in a club where this is one person speaking for
```
Comment (paked) [inline playbook/workshop_project_model.md]: `this`→`there`

File: playbook/workshop_project_model.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+
+They are designed so that by the end of every workshop, you can ship something
+tangible that you would want to show your friends.
+
+There should be a new workshop released every week.
+
+### Should everyone be working on the same workshop?
+
+People will want to work on different things and work at different rates.
+Let them.
```
Comment (paked) [inline playbook/workshop_project_model.md]: Perhaps add a note on how to introduce workshops to students?

File: playbook/workshop_project_model.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+Many people, when asked what project they want to work on, say they don't know.
+
+Doing a brainstorming session before this helps a lot.
+[Here is an example](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/ACTIVITIES.md#running-a-brainstorm-session)
+of how you might run a brainstorming session.
+
+### How do I run freeform project time?
+
+Here is a much more
+[comprehensive guide](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/ACTIVITIES.md#freeform-projects)
```
Comment (paked) [inline playbook/workshop_project_model.md]: Maybe move this into the hackedu/hackedu repository and make it a bit more specific to the code club environment

**Comment by MaxWofford** [comment]
@jonleung Would you like this merged still, or are you addressing the new comments?

**Comment by MaxWofford** [comment]
It looks like there are a few comments to address before this gets merged. Assigning to @jonleung

**Comment by zachlatta** [comment]
Any updates on this?

**Comment by zachlatta** [comment]
Ping.

**Comment by jonleung** [comment]
Closing this pull request because I decided not to put this in the playbook.

**Pull request closed without merging**